lib/engine/share_pool.rb

```diff
@@ -160,6 +160,10 @@ def transfer_shares(bundle, to_entity, spender: nil, receiver: nil, price: nil)
       num_shares.times { move_share(shares_of(corporation).first, owner) }
     end
 
+    def companies
```

Comment:
what is this for?

Reply:
I believe it is an artifact from Michael's start of a fix for the bug. Its not used anywhere in the fix, so will remove.

Reply:
Yeah I had added it in lib/engine/g_1846/share_pool.rb, I feel like something was breaking in ORs if a corporation was in receivership or something but I don't remember exactly. From a little bit of local testing taking out the function doesn't seem to break anything